openeuler-github-actions-ascend310--22.03.sp2/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Version: 20231107
+# Image name: quay.io/opencv-ci/opencv-openeuler-22.03.sp2-ascend310
+
+FROM openeuler/openeuler:22.03-lts-sp2
+
+RUN yum install -y \
+    git \
+    wget \
+    gcc \
+    g++ \
+    cmake \
+    make \
+    python-pip \
+    python3-devel \
+    xorg-x11-server-Xvfb \
+    ninja-build
```

Comment:
How about libs for image and video encoding / decoding (see below)? Are they in the base image `openeuler/openeuler:22.03-lts-sp2` already?

https://github.com/opencv-infrastructure/opencv-gha-dockerfile/blob/6b48bcb968f9150778d08d5eac390e15f9a38650/ubuntu-github-actions--22.04/Dockerfile#L28-L37

We may need them if testing dnn (load images or videos).

Reply:
Cannops module didn't depend on any of these libs. So I didn't install them. 
Let me install them and test with dnn test cases.